### ISSUE TITLE
File path should be configurable in System Settings

### ISSUE THREAD

**Issue by hansek** [BobRay/LogPageNotFound#2]
I would by great if there will be an option to define/override path to log file in System settings.

**Comment by BobRay** [comment]
You could just change this line in the plugin:

     $file = MODX_CORE_PATH . 'logs/pagenotfound.log';

**Comment by hansek** [comment]
Yes I know, but imho global settings is much cleaner way, easy update for everyone ;)

**Comment by BobRay** [comment]
OK, here you go: [commit] (https://github.com/BobRay/LogPageNotFound/commit/ab55a951b538e982ec526f47fb6277a7d54fcb3f)

You can create a System Setting called file_path, but using a property set would be better.
